interface/patient_file/manage_dup_patients.php

```diff
@@ -0,0 +1,298 @@
+<?php
+
+/*
+ * This tool helps with identifying and merging duplicate patients.
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    Rod Roark <rod@sunsetsystems.com>
+ * @copyright Copyright (c) 2017-2021 Rod Roark <rod@sunsetsystems.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+require_once("../globals.php");
+require_once("$srcdir/patient.inc");
+require_once("$srcdir/formatting.inc.php");
```

Comment:
remove the line above (already included in all scripts via composer.json)

Reply:
Done.